Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -418,8 +418,8 @@ InternalBootPickerEntryDraw (
   //
   // Draw the label horizontally centered.
   //
-  //ASSERT (Label->Width  <= BOOT_ENTRY_DIMENSION);
-  //ASSERT (Label->Height == BOOT_ENTRY_LABEL_HEIGHT);
+  //ASSERT (Label->Width  <= BOOT_ENTRY_DIMENSION * DrawContext->Scale);
```

Comment:
I believe we want a FIXME explaining why we cannot enable it (the limit is 340, but we cannot properly draw more).

Reply:
Right. Regarding size,
1) we can probably adapt offsets and such based of a Mac BootPicker screenshot, mine were just best guesses as I had no such available at the time.
2) Apple BootPicker implements label scrolling, I believe we want something like this too. A primitive version would just have a container GUI_OBJ with our maximum dimensions, and the actual label would be inside as wide as it wants to be. The inner object can then be slid freely to achieve the scrolling effect.